src/GRANULAR/pair_granular.cpp

```diff
@@ -1230,11 +1232,11 @@ void PairGranular::write_restart(FILE *fp)
         fwrite(&tangential_model[i][j],sizeof(int),1,fp);
         fwrite(&roll_model[i][j],sizeof(int),1,fp);
         fwrite(&twist_model[i][j],sizeof(int),1,fp);
-        fwrite(&normal_coeffs[i][j],sizeof(double),4,fp);
-        fwrite(&tangential_coeffs[i][j],sizeof(double),3,fp);
-        fwrite(&roll_coeffs[i][j],sizeof(double),3,fp);
-        fwrite(&twist_coeffs[i][j],sizeof(double),3,fp);
-        fwrite(&cut[i][j],sizeof(double),1,fp);
+        fwrite(normal_coeffs[i][j],sizeof(double),4,fp);
+        fwrite(tangential_coeffs[i][j],sizeof(double),3,fp);
```

Comment:
Why did the reference `&` disappear from some of the operations?

Reply:
because that is the bugfix. those are arrays not single values. see pair_granular.h

Reply:
Got it, thanks